phpstan.neon

```diff
@@ -10,6 +10,9 @@ parameters:
         # Exclude PHP Parser files
         - src/Parse/PHP/ArrayFile.php
         - src/Parse/PHP/ArrayPrinter.php
+        - src/Foundation/Console/KeyGenerateCommand.php
+        - src/Scaffold/GeneratorCommand.php
+    disableSchemaScan: true
```

Comment:
What does this do?

Reply:
It was suggested by phpstan at some point, but tried to remove it and looks like it's not required anymore.